spec/requests/pages/name_settings_spec.rb

```diff
@@ -0,0 +1,163 @@
+require "rails_helper"
+
+RSpec.describe "NameSettings controller", type: :request, feature_autocomplete_answer_types: true do
```

Comment:
```suggestion
RSpec.describe Pages::NameSettingsController, type: :request, feature_autocomplete_answer_types: true do
```

Its better to refer to the class name thats being used because rspec then knows what object you explicitly trying to test. It then opens us up to using to features such as using `described_class.new` instead of having to type out the class name every time and initialising it.

Reply:
@aliuk2012 Thanks Al, have done that for this and all the other settings controller specs 👍